common/scripts/poll-npm-package-published.mjs

```diff
@@ -0,0 +1,42 @@
+import { get } from "https";
+import process from "process";
+
+const POLL_INTERVAL_MS = 5 * 1000; // 5 seconds in millis
+const TOTAL_TIME_TO_POLL_MS = 5 * 60 * 1000; // 5 minutes in millis
+const MAX_RETRIES = TOTAL_TIME_TO_POLL_MS / POLL_INTERVAL_MS;
+
+const makeRequest = (url) => {
+  return new Promise((resolve, reject) => {
+    get(url, (response) => {
+      resolve(response);
+    }).end();
+  });
+}
+
+const sleep = (ms) => new Promise( res => setTimeout(res, ms));
+
+const main = async () => {
+  const url = process.argv[2];
```

Comment:
Huh, I expected the arguments to be `packageName` and `packageVersion`.

The script says it polls NPM, but the arguments allow polling anything ;)

Reply:
I never like faffing with argument names haha, but I have updated it to assume `@azure/communication-react` and to take in a version as the arg

Reply:
Unfortunately, we don't have `yargs` as a dependency here.

Look at the beauty of a fully fledged CLI args parser: https://github.com/Azure/communication-ui-library/blob/a1cc29e9a751b4a1a5f4b9039abd96b447363ff9/packages/react-composites/scripts/runBrowserTests.mjs#L139

Taking in just the version is fine. That's what I was going to do as well.